__tests__/app.test.js

```diff
@@ -361,4 +361,55 @@ describe("/api/articles/:article_id/comments", () => {
         expect(response.body.msg).toBe('Bad Request');
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/article/:article_id", () => {
+  test("GET:200 sends the article object with articel_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+      const date = new Date('2020-07-09 21:11:00') 
+        expect(response.body.article.length).toBe(1);
+        expect(response.body.article[0]).toEqual(expect.objectContaining({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: date.toISOString(),
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        }));          
+        });
+      });
+
+  test("GET:404 sends an NOT FOUND error if the article_id does not exist", () => {
+    return request(app)
+      .get("/api/articles/20")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("not found");
+      });
+  });
+
+  test("GET:200 sends an the article object with articel_id and with comments counted associated with this articel_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article[0].comment_count).toBe(11);
+      });
+  });
```

Comment:
It's great that you created a new test for the comment count - your testing acts as documentation, so this helps us see that a new feature has been developed